docs/typescript/activities.md

```diff
@@ -302,8 +302,8 @@ const { example } = proxyActivities<typeof activities>({
 });
 ```
 
-Without heartbeating, if your activity `StartToCloseTimeout` is 1 hour and the activity stalled or activity worker died, Temporal would have to wait out the 1 hour before retrying.
-But if you used the heartbeat API, set a `heartbeatTimeout` for 10 seconds, the absence of heartbeats in the `heartbeatTimeout` window would give the Server a signal that the activity has stalled and should be retried right away rather than at the end of the `StartToCloseTimeout`.
+Without Heartbeating, if your activity `StartToCloseTimeout` is 1 hour and the activity stalled or activity worker died, Temporal would have to wait out the 1 hour before retrying.
+But if you used the Heartbeat API, set a `heartbeatTimeout` for 10 seconds, the absence of heartbeats in the `heartbeatTimeout` window would give the Server a Signal that the activity has stalled and should be retried right away rather than at the end of the `StartToCloseTimeout`.
```

Comment:
In this case, Temporal Signals aren't involved, right? Perhaps something like the following?
```suggestion
But if you used the Heartbeat API, set a `heartbeatTimeout` for 10 seconds, the absence of Heartbeats in the `heartbeatTimeout` window would inform the Server that the Activity has stalled and should be retried right away rather than at the end of the `StartToCloseTimeout`.
```